<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundsPlayer</class>
 <widget class="QMainWindow" name="SoundsPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
border: 1px solid rgb(0, 150, 200);
border-radius:4px;
border-image: url(:/icon/res/777.jpg);
}</string>
   </property>
   <widget class="QLabel" name="lbl_song">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>260</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>无正在播放歌曲</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;关闭程序&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icon/res/close.png</normaloff>:/icon/res/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="play_curr">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>466</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icon/res/play1_hover1.png</normaloff>:/icon/res/play1_hover1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="play_next">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>476</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icon/res/next_hover1.png</normaloff>:/icon/res/next_hover1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="play_mode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>39</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>顺序</string>
    </property>
    <property name="iconSize">
     <size>
      <width>39</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="play_prev">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>476</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icon/res/prev_hover1.png</normaloff>:/icon/res/prev_hover1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
